### DISCUSSION TITLE
Can't Include TorchSharp.Jit for loading and inference on TorchScript module

### DISCUSSION THREAD

**Discussion by Blitzarcane** [xamarin/TorchSharp#893] [Q&A]
I'm using the TorchSharp NuGet Package 0.99.1 with corresponding TorchVision and Cuda backend package. I have a TorchScript module and I'm looking to do Jit.Module.Load as in this test: https://github.com/dotnet/TorchSharp/blob/28ef705c518b80fe9b17b30c573b84f78e594c6b/test/TorchSharpTest/TorchSharp/TorchSharp.cs#L220-L228

I'm not able to make any progress since TorchSharp.Jit appears to be missing. Is this functionality still available?

**Comment by NiklasGustafsson** [comment]
It should be `TorchSharp.torch.jit.load()`
